### ISSUE TITLE
undefined method `enc' for #<Gibberish::AES:0x00000006b32f80>

### ISSUE THREAD

**Issue by samuelralak** [ahoward/middleman-gibberish#7]
Hi,

I am trying to password protect my slate documentation using gibberish but each time I run **bundle exec middleman build** I get the following error:

**undefined method `enc' for #<Gibberish::AES:0x00000006b32f80>**

I honestly have no idea what the cause for this is. Please advise

**Comment by telephant00** [comment]
I am getting this same error, any info would be most appreciated! :)

**Comment by tchret** [comment]
+1

**Comment by forcefulhandshake** [comment]
Me too, any ideas on what is causing this?

**Comment by mashdot** [comment]
Anyone ever found out what's causing this?

**Comment by brycehewett** [comment]
+1

**Comment by mdoyle13** [comment]
I experienced the same issue so I forked the repo and pulled it down to work on locally.
After pointing my Gemfile at the local repo it works as it should

```gem 'middleman-gibberish', path: '/path/middleman-gibberish```

**Comment by brycehewett** [comment]
Yeah, it looks like the hosted gem is out of date. https://rubygems.org/gems/middleman-gibberish
You can also fix this by pointing directly to the git repo instead of the hosted gem.
`gem 'middleman-gibberish', :git => 'https://github.com/ahoward/middleman-gibberish.git'`
